pages/common/virtualenvwrapper.md

```diff
@@ -0,0 +1,32 @@
+# virtualenvwrapper
+
+> Group of simple wrapper commands for Python's `virtualenv` tool.
+> More information: <http://virtualenvwrapper.readthedocs.org>.
+
+- Create a new Python `virtualenv` in `$WORKON_HOME`:
+
+`mkvirtualenv {{virtualenv_name}}`
+
+- Create a `virtualenv` for a specific Python version:
+
+`mkvirtualenv --python {{/usr/local/bin/python2.7}} {{virtualenv_name}}`
```

Comment:
Perhaps we could use `python3` as an example here? Python2 was discontinued back in January.